lib/deaf_grandma.rb

```diff
@@ -19,9 +19,16 @@ def run!
 
 
   def speak(input)
-
-    #Implement your code here <<<<<<<<<
-
+    if input == "Bye!"
```

Comment:
One other note: The requirements state that Grandma doesn't respond to you saying "Bye!" until the third time, and that's when she says "See you later sonny". The way this code (and the tests) are written now, she says "See you later sonny" the first time you say bye.